lightning/src/routing/router.rs

```diff
@@ -784,6 +800,12 @@ where L::Target: Logger {
 					// Verify the liquidity offered by this channel complies to the minimal contribution.
 					let contributes_sufficient_value = available_value_contribution_msat >= minimal_value_contribution_msat;
 
+
+					// Do not consider candidates that exceed the maximum total cltv expiry limit.
+					let max_total_cltv_expiry_delta = payee.max_total_cltv_expiry_delta;
+					let hop_total_cltv_delta = $next_hops_cltv_delta + $directional_info.cltv_expiry_delta as u32;
```

Comment:
Grr, sorry, this should be a checked_add, otherwise I believe it can overflow (which panics in debug)

Reply:
Alright, is addressed by https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/f4e57f9ff08d930d58654ad8be1706e14f4d687c.